meilisearch-core/benches/search_benchmark.rs

```diff
@@ -15,7 +15,7 @@ use meilisearch_core::settings::{Settings, SettingsUpdate};
 use meilisearch_schema::Schema;
 use serde_json::Value;
 
-use criterion::{criterion_group, criterion_main, Criterion, BenchmarkId};
+use criterion::{black_box, criterion_group, criterion_main, Criterion, BenchmarkId};
```

Comment:
hello @neeldug  could you revert this line?

Reply:
Yep, my bad, was trying out black_box for more accurate benchmarking forgot to remove it from the PR.